com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -117,108 +79,58 @@ internal void __endSendServerRpc(NetworkSerializer serializer, uint rpcMethodId,
 #endif
         }
 
-#pragma warning disable IDE1006 // disable naming rule violation check
-        // RuntimeAccessModifiersILPP will make this `protected`
-        internal NetworkSerializer __beginSendClientRpc(uint rpcMethodId, ClientRpcParams clientRpcParams, RpcDelivery rpcDelivery)
-#pragma warning restore IDE1006 // restore naming rule violation check
+        public unsafe void SendClientRpc(ref FastBufferWriter writer, uint rpcMethodId, ClientRpcParams sendParams, RpcDelivery delivery)
```

Comment:
`ClientRpcParams sendParams` is not `sendParams`, it's still `rpcParams` with `.Send` inside it (same applies to `ServerRpcParams` above).
Either pass the inner `ClientRpcParams.Send` or just call this `rpcParams` as it used to be.

Reply:
Done.